.fixtures.yml

```diff
@@ -1,6 +1,7 @@
 fixtures:
   repositories:
     stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
+    transition: "https://github.com/puppetlabs/puppetlabs-transition.git"
```

Comment:
Needed because otherwise we would have duplicate resource declarations on platforms that need to do the package remove/install dance to upgrade (Solaris/SLES/AIX)